xlwings/main.py

```diff
@@ -482,7 +483,7 @@ def macro(self, name):
         .. code-block:: vb.net
 
         Function MySum(arr() as integer)
-            ''' code here
+            # code here
```

Comment:
given this is VBA, `'` would make more sense?

Reply:
Yes I thought so, but the formatting at https://docs.xlwings.org/en/stable/api.html#xlwings.App.macro was off and I thought perhaps the ‘#’ was giving the formatting error.

Reply:
I see, no, it's the missing indentation.